containers/release-candidate/src/daffodil-release-candidate

```diff
@@ -300,6 +301,23 @@ case $PROJECT_REPO in
     yarn package
     cp *.vsix $DAFFODIL_RELEASE_DIR/bin/
 
+    echo "Building Extension Docs..."
+    git clone https://github.com/apache/daffodil-vscode.wiki.git .wiki
+    for markdown_file in $(echo "\
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Development \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Roadmap \
+      Home"
```

Comment:
Can we automate getting the list of files? Seems it would be difficult to remember to update this script every time a new wiki page is created

Reply:
Yes, and you can get the list of files quite easily using globbing like this,

```shell
cd .wiki
for markdown_file in *.md; do
  pandoc -t html -f markdown -o "../${markdown_file}.html" "${markdown_file}"
done
```